src/02-crm/contacts.apib

```diff
@@ -135,6 +135,7 @@ Get details for a single contact.
             + added_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
             + `web_url`: `https://app.teamleader.eu/contact_detail.php?id=cde0bc5f-8602-4e12-b5d3-f03436b54c0d` (string)
+            + `national_identification_number`: BE 9823 (string)
```

Comment:
PP: it would be better to move this property somewhere after IBAN and BIC, before the more "preferency" properties. `web_url` is a metadata type of property so I wouldn't return more user provided entity properties after that one.